rclpy/rclpy/time.py

```diff
@@ -41,21 +51,22 @@ def __init__(
         self._time_handle = _rclpy.rcl_time_point_t(total_nanoseconds, clock_type)
 
     @property
-    def nanoseconds(self):
+    def nanoseconds(self) -> int:
+        """:return: the total number of nanoseconds since the clock's epoch."""
         return self._time_handle.nanoseconds
 
-    def seconds_nanoseconds(self):
+    def seconds_nanoseconds(self) -> Tuple[int, int]:
         """
-        Get time as separate seconds and nanoseconds components.
+        Get time separated into seconds and nanoseconds components.
 
         :returns: 2-tuple seconds and nanoseconds
-        :rtype: tuple(int, int)
```

Comment:
I notice that here you've removed `rtype`, but in some other places you added it.  This is again a place I'd hope we can be consistent, either always having the `rtype`, or always omitting it.  That said, there may be Python standards here I don't know about, so take this comment with a grain of salt.

Reply:
I went with what looked best in the generated documentation. Type annotations worked fine here, but not for returning `builtin_interfaces.msg.Time` because sphinx shortens it to `Time`, so I added `:rtype:` there.